kyaml/runfn/runfn.go

```diff
@@ -518,3 +530,20 @@ func (r *RunFns) ffp(spec runtimeutil.FunctionSpec, api *yaml.RNode, currentUser
 
 	return nil, nil
 }
+
+func getWorkingDirectory(wd string) (string, error) {
```

Comment:
Wdyt about moving these sanity checks to `(c *Filter) Run` so that we can be sure they'll be run no matter how the filter was instantiated?

Reply:
moved